src/DynamoCore/Configuration/Configurations.cs

```diff
@@ -116,6 +116,11 @@ public class Configurations
         /// </summary>
         public static readonly double ZoomIncrement = 0.05;
 
+        /// <summary>
+        /// Zoom Threshold Level 
```

Comment:
Would you be more specific here, e.g. `Global zoom threshold level for node overlay to apply`

Reply:
Yup, done! Thank you